terraform/variables.tf

```diff
@@ -209,3 +228,9 @@ variable "net_profile_service_cidr" {
   default     = "10.0.0.0/16"
   type        = string
 }
+
+variable "build_backstage" {
+  description = "Flag to control whether Backstage-related components are built"
+  type        = bool
+  default     = true
```

Comment:
This adds a lot more things which could go wrong and slows down the deployment.  Let's set to `false` by default.  From what I can see, users should be able to set the var to `true` and then `terraform apply` to make it happen.